dashboard/db/migrate/20180131192613_add_timestamps_to_featured_projects.rb

```diff
@@ -0,0 +1,6 @@
+class AddTimestampsToFeaturedProjects < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :featured_projects, :created_at, :integer
+    add_column :featured_projects, :featured_at, :datetime
```

Comment:
Just confirming we no longer care about the unfeatured timestamp?

Reply:
good catch Andrew! we do need `unfeatured_at` to be added here. It looks like the migration and the model/schema got out of sync in the process of rolling the migrations backward/forward.